organizations/admin/classes/domain/OrgEx.php

```diff
@@ -0,0 +1,333 @@
+<?php
+
+/*
+**************************************************************************************************************************
+** CORAL Licenses Module v. 1.2
+**
+** Copyright (c) 2010 University of Notre Dame
+**
+** This file is part of CORAL.
+**
+** CORAL is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+**
+** CORAL is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License along with CORAL.  If not, see <http://www.gnu.org/licenses/>.
+**
+**************************************************************************************************************************
+*/
+
+class OrgEx extends DatabaseObject {
+
+
+  protected function overridePrimaryKeyName() {}
+
+
+    
+  public static function setSearch($search) {
+  $config = new Configuration;
+    echo var_dump($search);
+    echo "we get into setSeach at some point$ ".$config->settings->defaultsort."$\n";
+    if ($config->settings->defaultsort) {
+      $orderBy = $config->settings->defaultsort;
+    } else {
+      $orderBy = "R.createDate DESC, TRIM(LEADING 'THE ' FROM UPPER(R.shortName)) asc";
+    }
+    $defaultSearchParameters = array(
+    "orderBy" => $orderBy,
+    "page" => 1,
+    "recordsPerPage" => 25,
+    );
+    foreach ($defaultSearchParameters as $key => $value) {
+      if (!isset($search[$key])) {
+        $search[$key] = $value;
+      }
+    }
+    foreach ($search as $key => $value) {
+      $search[$key] = trim($value);
+    }
+    CoralSession::set('OrgSearch', $search);
```

Comment:
This doesn't work for me, since CoralSession isn't defined as a class for the organizations module. Copying over CoralSession.php from `resources/admin/classes/common` would fix this.